calico_node/Makefile

```diff
@@ -42,19 +42,6 @@ KUBE_CONTROLLERS_VER ?= $(V_KUBE_CONTROLLERS)
 LIBNETWORK_PLUGIN_VER ?= $(V_LIBNETWORK_PLUGIN)
 TYPHA_VER ?= $(V_TYPHA)
 
-$(info RELEASE_STREAM=$(RELEASE_STREAM))
-$(info CALICO_VER=$(CALICO_VER))
-$(info CALICO_GIT_VER=$(CALICO_GIT_VER))
-$(info BIRD_VER=$(BIRD_VER))
-$(info CALICOCTL_VER=$(CALICOCTL_VER))
-$(info CONFD_VER=$(CONFD_VER))
-$(info CNI_VER=$(CNI_VER))
-$(info FELIX_VER=$(FELIX_VER))
-$(info GOBGPD_VER=$(GOBGPD_VER))
-$(info KUBE_CONTROLLERS_VER=$(KUBE_CONTROLLERS_VER))
-$(info LIBNETWORK_PLUGIN_VER=$(LIBNETWORK_PLUGIN_VER))
-$(info TYPHA_VER=$(TYPHA_VER))
```

Comment:
I like having these printed on every make call, and see no benefit in removing them.

Reply:
Printing component versions for unrelated targets in a hard-to-read format is not desirable. If you only see information related to your action, mistakes are less likely.

If we dump 12 version strings every time you make `clean` or `vendor`, you'll stop noticing those strings - even when you're cutting a release. 

The other thing this particular version gets you is the ability to run `make print-versions` which is, as I've mentioned a few times, _extremely_ helpful while cutting a release.

If you still have questions/comments about this, let's find a few minutes to chat in person.

Reply:
@neiljerram - I decided this point isn't worth debating anymore so I restored the version printing for all targets. I preserved the "pretty print" since it helps digest the information as it scrolls past.